src/tracer.js

```diff
@@ -203,7 +203,8 @@ module.exports.postTrace = function postTrace(traceObject) {
     utils.debugLog(`trace: ${JSON.stringify(traceObject, null, 2)}`);
     return session.post(
         config.getConfig().traceCollectorURL,
-        traceObject
+        traceObject,
+        { headers: { Authorization: `Bearer ${config.getConfig().token}` } }
```

Comment:
That's for future compatibility?
Isn't there a more suitable way to add authorization info without explicitly changing the HTTP headers?

Reply:
Yes, that's for the future.
Authorization header is probably the most common way to pass tokens :)